python/tk_softimage/menu_generation.py

```diff
@@ -103,7 +103,9 @@ def _add_context_menu(self):
         ctx_name = str(ctx)
 
         # create the sub menu object
-        ctx_menu = self._menu_handle.AddSubMenu(ctx_name)
+        # the label expects a unicode object so we cast it to support when the context may 
+        # contain info with non-ascii character
+        ctx_menu = self._menu_handle.AddSubMenu(ctx_name.decode("utf-8"))
```

Comment:
AddSubMenu() wraps the SoftImage api so I'm thinking this decode might make more sense here:
https://github.com/shotgunsoftware/tk-softimage/blob/master/plugins/shotgun/Application/Plugins/menu.py#L126
where it calls self._si_AddSubMenu(name)